sources/packages/web/src/assets/css/base.scss

```diff
@@ -79,15 +80,31 @@ html * {
   border-color: $badge-border-error !important;
 }
 
-/** Style to align records per page with arrows in data table. **/
+// Data table styles
```

Comment:
mm, we add a full stop after each sentence in our comments. we review other comments too

Reply:
@ann-aot Thanks Ann - can you give me an example what you mean? Are you referring to the commenting style?

Reply:
example, 
![image](https://user-images.githubusercontent.com/77353155/204353252-ca076d48-5f56-4872-93b3-907a41d18e05.png)
![image](https://user-images.githubusercontent.com/77353155/204353416-d18825d9-47b2-4fe6-826c-7790c1a88857.png)
period `(.)` at the end of a sentance